BookingProject/BookingWebAPI/BookingWebApi.Application/Decorators/SignInManagerDecorator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookingWebApi.Application.Decorators
+{
+    public class SignInManagerDecorator<T> : ISignInManagerDecorator<T> where T : class
```

Comment:
Move implementation to infrastructure layer, because Identity and DB interaction is Infrastructure concerns